include/onnxruntime/core/graph/indexed_sub_graph.h

```diff
@@ -36,6 +36,9 @@ struct IndexedSubGraph {
     NodeAttributes attributes;         ///< Attributes of customized SubGraph/FunctionProto.
 
     std::string doc_string;  ///< Doc string of customized SubGraph/FunctionProto.
+    
+    /** Type and shape inference function that should be defined for the fused node */
+    std::function<void (ONNX_NAMESPACE::InferenceContext&)> inference_function;
```

Comment:
nit: comment is misleading. 'should be defined' -> 'can optionally be defined' 

'inference_function' is a bit vague as well. can we make it 'type_and_shape_inference_function'?

Reply:
Fixed